packages/@lightningjs/ui-components/src/components/TabBar/Tab.d.ts

```diff
@@ -29,7 +29,7 @@ type TabStyle = SurfaceStyle & {
   contentColor: Color;
   iconMarginRight: number;
   iconSize: number;
-  noTitlePaddingX: number;
+  paddingXNoTitle: number;
```

Comment:
keep the old one here too and add the `@deprecated` comment above it. Check out Button.d.ts for reference

Reply:
regarding this, when possible we should add a message for some context around the deprecation - whether that's "this went away cause this" and/or "use x instead"